epubmaker_preprocessing.js

```diff
@@ -86,7 +86,7 @@ fs.readFile(file, function editContent (err, contents) {
     mySibling.remove();
   });
 
-  // turn links into real hyperlinks
+    // turn links into real hyperlinks
```

Comment:
Doesn't really matter, but this tab is unneeded, and since there are no functional edits in this file, possibly accidental?